uisdk/src/main/res/values/strings.xml

```diff
@@ -387,4 +387,8 @@
     <string name="kh_uisdk_loyalty_points_earned_for_trip">This ride earns you %1$s points</string>
     <string name="kh_uisdk_loyalty_pre_auth_not_enough_points">Your balance is insufficient</string>
     <string name="kh_uisdk_loyalty_pre_auth_not_allowed_to_burn">You are not allowed to burn points</string>
+
+    <!--  Validity  -->
```

Comment:
Have we added this values to Phrase.com? I am wondering why the IT, DE and NL translations are missing?